app/pages/delete-order/delete-order-confirmation/ui-tests/general.ui.test.js

```diff
@@ -0,0 +1,113 @@
+import nock from 'nock';
+import { ClientFunction, Selector } from 'testcafe';
+import { extractInnerText } from 'buying-catalogue-library';
+import content from '../manifest.json';
+import { nockAndErrorCheck, setState, authTokenInSession } from '../../../../test-utils/uiTestHelper';
+import { orderApiUrl } from '../../../../config';
+
+const pageUrl = 'http://localhost:1234/order/organisation/order-id/delete-order/confirmation';
+
+const orderDescriptionMock = 'desc';
+
+const mocks = ({ postRoute = false }) => {
+  nock(orderApiUrl)
+    .get('/api/v1/orders/order-id/sections/description')
+    .reply(200, { description: orderDescriptionMock });
+
+  if (postRoute) {
+    nock(orderApiUrl)
+      .delete('/api/v1/orders/order-id')
+      .reply(204);
+  }

```

Comment:
do we need this nock or the post-route for the delete-order-confirmation page?

Reply:
No sorry, we dont!